frontend/src/App.tsx

```diff
@@ -9,6 +9,7 @@ import ScrollToTop from './components/ScrollToTop';
 import { ProgressBarStyle } from './components/ProgressBar';
 import NotistackProvider from './components/NotistackProvider';
 import MotionLazyContainer from './components/animate/MotionLazyContainer';
+import { SnackbarUtilsConfigurator } from './utils/snackbarUtil';
```

Comment:
You don't need a new snackbar for this. You can just import useSnackbar.

And use enqueueSnackbar.

Can just do a search for to see how it works.

Reply:
useSnackbar requires a SnackbarProvider, which is only rendered in App.tsx. The service worker registration can only be done in index.tsx, hence I had to use this SnackbarUtilsConfigurator to place a component in the tree that will be able to access the SnackbarProvider.